examples/light-js/index.html

```diff
@@ -10,18 +10,37 @@
   </head>
 
   <body>
-    <div><h2>Status</h2></div>
+    <div>
+      <h2>Status</h2>
+    </div>
     <div id="status"></div>
 
-    <div><h2>Local Peer Id</h2></div>
+    <div>
+      <h2>Local Peer Id</h2>
+    </div>
     <div id="peer-id"></div>
 
-    <div><h2>Remote Peer Id</h2></div>
-    <div id="remote-peer-id"></div>
-
-    <label for="remote-multiaddr">Remote peer's multiaddr</label>
-    <input id="remote-multiaddr" type="text" value="" />
+    <div>
+      <h2>Select Peer</h2>
+    </div>
+    <div>
+      <button id="getPeersButton" type="button">Refresh Peers</button>
+      <select id="peer-select">
+        <option value=""></option>
+      </select>
+    </div>
+    <div>
+      <label for="remote-multiaddr">Remote peer's multiaddr</label>
+    </div>
+    <div>
+      <input id="remote-multiaddr" type="text" value="" style="width: 100%" />
```

Comment:
nit: let's add some styling - this field is soooo long :)

Reply:
Added a right margin to parent div and made the maximum width for the input 900px; this is about as wide it needs to be to display the entire multiaddr